day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,27 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    @toppings = toppings
```

Comment:
Right now you're reassigning your toppings, how would you change it so that you're adding to the existing array?